buildSrc/build.gradle.kts

```diff
@@ -159,7 +159,7 @@ fun Project.applyGroovyProjectConventions() {
         }
     }
 
-    tasks.withType<GroovyCompile>().configureEach {
+    tasks.withType<GroovyCompile>() {
```

Comment:
❓ My understanding is that `tasks.withType<GroovyCompile> { ... }` will eagerly realize all `GroovyCompile` tasks in order to configure them while `tasks.withType<GroovyCompile>().configureEach { ... }` wouldn't, is that correct?

Reply:
That's my understanding as well, so I think `.configureEach` should be kept.